src/Illuminate/Foundation/Console/EventListCommand.php

```diff
@@ -21,14 +22,31 @@ class EventListCommand extends Command
      */
     protected $description = "List the application's events and listeners";
 
+    /**
+     * The table headers for the command.
+     *
+     * @var array
+     */
+    protected $headers = ['Event', 'Listeners'];
+
     /**
      * Execute the console command.
      *
      * @return mixed
      */
     public function handle()
     {
-        $this->table(['Event', 'Listeners'], $this->getEvents());
+        $events = $this->getEvents();
+
+        if (empty($events)) {
+            if ($this->isSearching()) {
+                return $this->error('Your application doesn\'t have any events matching the given criteria.');
+            }
+
+            return $this->error('Your application doesn\'t has any events, listeners.');
```

Comment:
Typo....should be...

```php
return $this->error('Your application doesn\'t have any event listeners.');
```

Reply:
Thank you for your comment! This command still lists an event to the table even it doesn't has any listeners. The error message is only printed when no event is found, so I think the right message may be:

```php
'Your application doesn\'t have any events.'
```

But Mr.Otwell always modifies the message of an accepted commit if he needs. Therefore, let him decide finally.

Reply:
> 'Your application doesn\'t have any events.'

If you define the word "event" as a synonym for "event class" -> might be true then.
If you define the word "event" as a the name of an event that can be passed to the `event()` helper function - for example `event('event-'.time())` then the statement is not valid, because you cannot automatically detect event names.

Reply:
> But Mr.Otwell always modifies the message of an accepted commit if he needs. Therefore, let him decide finally.

Indeed, such changes have been made:
https://github.com/laravel/framework/commit/cde1c5d8b38a9b040e70c344bba82781239a0bbf